Automattic/wp-calypso
# CRO i.2: create Jetpack Product Slide Out Card

#### Changes proposed in this Pull Request

* Create the `JetpackProductSlideOutCard` component which, in i2, is used inside a `JetpackProductCardAlt` component.

#### Notes

* Around 300 lines were updated only to fix the `eslint(wpcalypso/no-package-relative-imports)` warning.

#### Testing instructions

* Run this PR (Calypso Blue).
* Visit `http://calypso.localhost:3000/devdocs/design`.
* Search for `JetpackProductSlideOutCard`.
* Observe the two examples provided (Jetpack Backup Daily and Jetpack Scan).
* Make sure they look as per design.

Fixes 1196341175636977-as-1198514641268282

#### Demo
![image](https://user-images.githubusercontent.com/3418513/97064064-e741b000-1579-11eb-9224-07de1c070743.png)
